docs/synthetic-tokens/what-are-synthetic-assets.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Synthetic tokens
+sidebar_label: Synthetic tokens
+---
+
+## What are synthetic tokens?
+
+Synthetic tokens are collateral-backed tokens whose value fluctuates depending on the tokens’ reference index. Synthetic tokens blend features of prediction markets, futures markets, and collateralized loans.
+
+Some examples of synthetic tokens include:
```

Comment:
I don't like repeating redundant content here. I think we need to come up with some different content for this or the one on the "Welcome to UMA" page, or just remove one